sormas-ui/src/main/java/de/symeda/sormas/ui/location/LocationEditForm.java

```diff
@@ -83,7 +88,7 @@
 		//XXX #1620 are the divs needed?
 		divs(
 			fluidRowLocs(LocationDto.ADDRESS_TYPE, LocationDto.ADDRESS_TYPE_DETAILS, ""),
-			fluidRowLocs(LocationDto.COUNTRY, "", ""),
+			fluidRowLocs(LocationDto.COUNTRY, LocationDto.SUB_CONTINENT, LocationDto.CONTINENT),
```

Comment:
Order should be reversed (Continent -> Subcontinent -> Country); since that would look weird when the continent and subcontinent fields are hidden, I'd suggest to move them to a separate line above the country field